aicsimageio/writers/ome_tiff_writer.py

```diff
@@ -166,11 +166,16 @@ def save(
                 channel_colors=channel_colors,
                 dimension_order=dimension_order,
             )
+            xml = self.omeMetadata.to_xml().encode()
+        elif type(ome_xml) == str:
```

Comment:
Change to `elif isinstance(ome_xml, str):`

Reply:
i googled this and they seemed interchangeable. I'll change it but I'm curious-  do you know what the difference is?

Reply:
I found my answer.  isinstance can handle inheritance so it can cover more potential cases